configs/containers/specs/jedi-ci.yaml

```diff
@@ -34,25 +34,24 @@
     netcdf-fortran@4.6.1,
     parallelio@2.6.2 +pnetcdf,
     parallel-netcdf@1.12.3,
-    py-eccodes@1.5.0,
-    py-f90nml@1.4.3,
-    py-gitpython@3.1.40,
-    py-h5py@3.11.0,
-    py-numpy@1.23.5,
-    py-pandas@1.5.3,
+    py-eccodes,
+    py-f90nml,
+    py-gitpython,
+    py-h5py,
+    py-numpy,
+    py-pandas,
     py-pip,
-    py-pyyaml@6.0,
-    py-scipy@1.12.0,
-    py-shapely@1.8.0,
-    py-xarray@2023.7.0,
-    sp@2.5.0,
```

Comment:
Time to remove this, and use `ip@5` instead. See https://github.com/GEOS-ESM/GSIbec/pull/66 for a boilerplate PR for applications using `sp` at the moment.